elastic/kibana
# [Synthetics] Support lightweight project monitors

## Summary

Testing
--
1. Navigate to Uptime and create an Synthetics API key
2. Checkout the corresponding Synthetics agent PR and follow the instructions https://github.com/elastic/synthetics/pull/593
3. Navigate to monitor management to confirm lightweight monitors have been created appropriately.

Tested against following input

  ```
const schema = {
    project: 'test',
    keep_stale: false,
    monitors: [
      {
        locations: [],
        privateLocations: ['fleet-server-policy'],
        type: 'http',
        enabled: false,
        id: 'my-monitor-1',
        name: 'My Monitor 1',
        urls: ['http://localhost:9200'],
        schedule: 1,
        timeout: '16s',
        content: '',
      },
      {
        locations: [],
        privateLocations: ['fleet-server-policy'],
        type: 'http',
        enabled: false,
        id: 'my-monitor-2',
        name: 'My Monitor 2',
        urls: ['http://localhost:9200'],
        schedule: 60,
        timeout: '16s',
        content: '',
      },
    ],
  };
```
